content/en/docs/concepts/cluster-administration/dra.md

```diff
@@ -96,34 +98,36 @@ client-go configuration within `kube-controller-manager` are critical.
 
 The specific values to tune your cluster to depend on a variety of factors like
 number of nodes/pods, rate of pod creation, churn, even in non-DRA environments;
-see the [SIG-Scalability README on Kubernetes scalability
+see the [SIG Scalability README on Kubernetes scalability
  thresholds](https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/thresholds.md)
 for more information. In scale tests performed against a DRA enabled cluster
 with 100 nodes, involving 720 long-lived pods (90% saturation) and 80 churn pods
 (10% churn, 10 times), with a job creation QPS of 10, `kube-controller-manager`
 QPS could be set to as low as 75 and Burst to 150 to meet equivalent metric
 targets for non-DRA deployments. At this lower bound, it was observed that the
 client side rate limiter was triggered enough to protect apiserver from
-explosive burst but was is high enough that pod startup SLOs were not impacted.
+explosive burst but was high enough that pod startup SLOs were not impacted.
 While this is a good starting point, you can get a better idea of how to tune
 the different components that have the biggest effect on DRA performance for
-your deployment by monitoring the following metrics.
+your deployment by monitoring the following metrics. For more information on all
+the stable metrics in Kubernetes, see the [Kubernetes Metrics
+Reference](/docs/reference/generated/metrics/).
 
 ### `kube-controller-manager` metrics
 
 The following metrics look closely at the internal ResourceClaim controller
 managed by the `kube-controller-manager` component.
 
-* Workqueue Add Rate: Monitor
-  `sum(rate(workqueue_adds_total{name="resource_claim"}[5m]))` to gauge how
-  quickly items are added to the ResourceClaim controller.
+* Workqueue Add Rate: Monitor {{< highlight promql
```

Comment:
@lmktfy in https://github.com/kubernetes/website/pull/51722#discussion_r2257921371 and others like it, you had asked about linking inline direct to the relevant metric in the metric reference page, but I couldn't confirm that the reference doc headings in https://kubernetes.io/docs/reference/instrumentation/metrics/ supported direct linking via url fragment, plz help me if I am missing that 😇

Reply:
I added it in [whats next](https://deploy-preview-51818--kubernetes-io-main-staging.netlify.app/docs/concepts/cluster-administration/dra/#what-s-next) section and in the last sentence of the intro on metrics (below), let me know if it should be more visible or just more overall.
<img width="699" height="325" alt="Screenshot 2025-08-07 at 11 21 44 PM" src="https://github.com/user-attachments/assets/38086782-9c37-4536-8644-c119a50f53d9" />